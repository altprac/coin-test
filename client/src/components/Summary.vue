<template>
   <div class="row">
     <div class="col text-center">
      <h1>{{total}}</h1>
      Your current balance {{VUE_APP_ENABLE_MAINTENANCE}}
      </div>
  </div>
</template>

<script>
export default {
  props: ['total']
}
</script>

<style>
</style> 
