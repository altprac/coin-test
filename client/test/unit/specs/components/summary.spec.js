import { shallowMount } from '@vue/test-utils'
import Summary from '@/components/Summary'

describe('Summary.vue', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(Summary, { // Create a shallow instance of the component
      propsData: {
        total: '$100'
      }
    })
  })
  it('has the expected html structure', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
