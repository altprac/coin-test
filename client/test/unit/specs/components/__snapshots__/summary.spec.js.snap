// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Summary.vue has the expected html structure 1`] = `
<div
  class="row"
>
  <div
    class="col text-center"
  >
    <h1>
      $100
    </h1>
    
    Your current balance
    
  </div>
</div>
`;
